app/src/main/java/com/github/se/eventradar/ui/BottomNavigationMenu.kt

```diff
@@ -24,25 +36,65 @@ fun BottomNavigationMenu(
     modifier: Modifier = Modifier,
 ) {
   BottomNavigation(
-      backgroundColor = MaterialTheme.colorScheme.secondaryContainer,
+      backgroundColor = MaterialTheme.colorScheme.surfaceContainer,
       modifier = modifier,
   ) {
     tabList.forEach { tab ->
+      val boxModifier =
+          if (tab == selectedItem) {
+            Modifier.clip(shape = RoundedCornerShape(50))
+                .background(MaterialTheme.colorScheme.primary)
+                .width(58.dp)
+                .height(36.dp)
+          } else Modifier
+
+      val labelText = if (selectedItem == tab) stringResource(tab.textId) else ""
+
       BottomNavigationItem(
-          icon = {
-            Icon(
-                painter = painterResource(id = tab.icon),
-                contentDescription = null,
-                modifier = Modifier.width(24.dp).height(24.dp),
-            )
+          icon = { NavBarIcon(tab, boxModifier) },
+          label = {
+            Text(
+                text = labelText,
+                color = MaterialTheme.colorScheme.onSurfaceVariant,
```

Comment:
nit: You may want to change the color of the icon when selected so that it shows better.

![image](https://github.com/swent-sp-2024-event-radar/event-radar-app/assets/64320145/fe212e98-d586-4b6a-9a50-f087a6e61c61)